src/karmen_backend/server/routes/octoprintemulator.py

```diff
@@ -25,6 +25,80 @@ def version():
     )
 
 
+@app.route("/octoprint-emulator/api/settings", methods=["GET"])
+@cross_origin()
+def settings():
+    token = request.headers.get("x-api-key", None)
```

Comment:
Can we move the token check to a decorator? Some inspiration can be drawn from [this one](https://github.com/fragaria/karmen/blob/master/src/karmen_backend/server/routes/__init__.py#L49). The request and abort can be used in a global scope (that's how flask is designed), so it shouldn't be a problem.